vllm/model_executor/layers/mamba/ops/mamba_ssm.py

```diff
@@ -220,6 +287,9 @@ def selective_state_update(
     dst_state_batch_indices=None,
     pad_slot_id=PAD_SLOT_ID,
     out=None,
+    inplace_final_state=False,
+    num_accepted_tokens=None,
+    cu_seqlens=None,
```

Comment:
Could you add a comment explaining these? Especially `inplace_final_state`, as I'm not clear on when to set it